tools/pre_ci.py

```
@@ -0,0 +1,275 @@
+        Returns:
+            bool: True if the command succeeded (exit code 0), False otherwise.
+        """
+        print(f"\n>>> [Step: {description}]")
+        try:
+            subprocess.run(command, check=True)
+            print(f"✅ {description} completed successfully.")
+            self.record_result(description, True)
+            return True
+        except subprocess.CalledProcessError as e:
+            print(f"\n❌ FATAL: '{description}' failed with exit code {e.returncode}")
+            self.record_result(description, False)
```

Ruff (0.15.11)

[warning] Boolean positional value in function call

(FBT003)